iquip/apps/logger.py

```diff
@@ -184,11 +228,11 @@ def addLog(self, content: str):
 
     @pyqtSlot()
     def checkToClear(self):
-        """Shows a confirmation frame for clearing logs."""
+        """Shows a confirmation frame for clearing logs in the loggerFrame."""
         logger.info("Tried to clear logs by clicking clear button")
         self.confirmFrame.show()
```

Comment:
I think we don't need this anymore since we log to the files.

Reply:
You mean not using `confirmFrame`?

Reply:
I understand that we can clear logs in screen right away since we are writing at file.
But I want to make sure before delete that feature.